.gitignore

```diff
@@ -31,4 +31,4 @@ pkg/**/._gometalinter
 vendor/
 .vscode/
 
-pkg/main/__debug_bin
\ No newline at end of file
+pkg/main/__debug_bin.DS_Store
```

Comment:
What is the rational for this change?

Reply:
@paulcarlton-ww  Will remove this might be by mistake.